src/Serialization/upgrades/0.11.3.jl

```diff
@@ -0,0 +1,120 @@
+################################################################################
+# Upgrade Summary
+# Types of the form `Vector{T}` are only serialized with backref when
+# `is_basic_serialization_type(T) == false`. If
+# `is_basic_serialization_type(T) == true` the type `Vector{T}` does not
+# serialize `T` in the entries, but instead as a part of the `Vector` serialization.
+# Instances of the form `Tuple{T}` with `is_basic_serialization_type(T) == true` 
+# are no longer serialized with backref, however these types were already storing
+# there types outside of the JSON array of entries
+
+push!(upgrade_scripts, UpgradeScript(
+    v"0.11.3", # version this script upgrades to
+    function (s::DeserializerState, dict::Dict)
+        # file comes from polymake
+        if haskey(dict, :_ns)
+            haskey(dict[:_ns], :polymake) && return dict
+        end
```

Comment:
This mixes two idioms and I had to read it twice to get it... I suggest to rewrite it either as this:
```suggestion
        if haskey(dict, :_ns) && haskey(dict[:_ns], :polymake)
           return dict
        end
```
or as this
```suggestion
        haskey(dict, :_ns) && haskey(dict[:_ns], :polymake) && return dict
```